routes/search/operations.js

```diff
@@ -0,0 +1,38 @@
+module.exports = [
+  {
+    name: 'docs',
+    displayName: 'Documentations',
+    query: `query FilterDocs($filter: String) {
+      docs(filter: $filter){
+        title
+        description
+        href
+        isTutorial
+        isApiDoc
+        isDevTutorial
+        isApiStructureDoc
+      }
+    }`,
+    partial: 'search/docs'
+  }, {
+    name: 'npmPackages',
+    displayName: 'NPM Packages',
```

Comment:
`npm` should always be lowercase. It's an exception to the rule. These display name strings should also ideally live in `locale.yml` so that can be localized.